### PR TITLE
More app-friendly event description

### PR DESCRIPTION
**Pull request by WarFollowsMe**
I suggest change a little bit descriptions of all events to make them more convenient for monitoring tools. 

There is a task to allow the user to configure filtering of incoming events by the value of their arguments. But it's problematic to do that when the event contains only an enumeration of data types and no argument names are specified. Now you have to add these names in the semi-handheld mode. I suggest to add an entry in square brackets with enumerated names to the event description where there are many arguments or these arguments of the same type. Right now a similar way of description is often used, but the names are described in normal brackets. Unfortunately, normal brackets are also used to describe additional information, which creates confusion when an argument name is automatically parsed.

In correcting the comments, I was guided by the following rules:
- if the type of argument fully describes its purpose, there is no need to specify the name;
```
/// Ethereum events from contracts.
Log(Log)
```
- if there is one argument in the event and it is necessary to specify its name, it is enough to find the corresponding word in the description and select it in square brackets;
```
/// A contract has been created at given [address].
Created(H160)
```
- if it is necessary to specify several argument names, then after the whole description we create an array of required names.
```
/// A deposit has been made at a given address. [sender, address, value]
BalanceDeposit(AccountId, H160, U256),
```

### CODE REVIEW

File: frame/assets/src/lib.rs [hunk 1]
```diff
@@ -230,11 +230,11 @@ decl_event! {
 		<T as Trait>::Balance,
 		<T as Trait>::AssetId,
 	{
-		/// Some assets were issued.
+		/// Some assets were issued. [asset_id, origin, total_supply]
```
Comment (bkchr) [inline frame/assets/src/lib.rs:233]: The second value is clearly not the origin.

File: frame/assets/src/lib.rs [hunk 2]
```diff
@@ -230,11 +230,11 @@ decl_event! {
...
 		Issued(AssetId, AccountId, Balance),
-		/// Some assets were transferred.
+		/// Some assets were transferred. [asset_id, from, to, amount]
 		Transferred(AssetId, AccountId, AccountId, Balance),
-		/// Some assets were destroyed.
+		/// Some assets were destroyed. [asset_id, origin, balance]
```
Comment (bkchr) [inline frame/assets/src/lib.rs:237]: Same

File: frame/assets/src/lib.rs (hunk 1 above)
Comment (WarFollowsMe) [reply frame/assets/src/lib.rs:233]: sender? I was just checking a references. Found this
```
Self::deposit_event(RawEvent::Issued(id, origin, total));
```

Comment (WarFollowsMe) [reply frame/assets/src/lib.rs:233]: or just `account` I think would be enough.

Comment (kianenigma) [reply frame/assets/src/lib.rs:233]: `owners` makes more sense to me.

**Comment by danforbes** [comment]
Thank you for tagging me, Kian :pray: If this is deemed to be a worthwhile idea, I think the final style (an array of names after the description) should be applied to _all_ event descriptions.

**Comment by WarFollowsMe** [comment]
I was searching for events in code by `decl_event!` string. Tried to fix everything I found. How else can I find the rest of the events?

**Comment by xlc** [comment]
https://github.com/paritytech/substrate/issues/5913 should solve this issue, but this is a good intermediate solution as I don't think #5913 is going to happen anytime soon.

**Pull request merged**